journal-of-universal-computer-science.csl

```diff
@@ -0,0 +1,529 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" and="text" delimiter-precedes-last="never" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
```

Comment:
Presumably the journal is English only, right? In that case, please add `default-locale="en-US"` to this line.